source/github-helpers/github-url.ts

```diff
@@ -36,16 +36,23 @@ export default class GitHubURL {
 	private disambiguateReference(
 		ambiguousReference: string[]
 	): {branch: string; filePath: string} {
-		const branch = ambiguousReference[0];
+		let branch;
+		let filePath;
+		if (ambiguousReference[1]?.includes('%2')) {
```

Comment:
FYI this ('%2) will only happen on the main tree page once you go into any subfile or folder it will change back to slashes

Reply:
The changes to this file should be reverted because: If the path has `%2F`, it's already not ambiguous and should not be passed to `disambiguateReference`

`a/b%2Fc` is unambiguously the name of the branch. Names won't have `%2F` in the filename (because it'd be represented as [a/b%252Fc`](https://github.com/sindresorhus/refined-github/blob/branch/file%252Ffile) in the URL) and `/` cannot be part of the filename.